README.md

```diff
@@ -21,3 +21,18 @@ The library is distributed using a modified BSD-style
 
 The official source code repository is located at
 https://github.com/avrdudes/avr-libc/
+
+### Building AVR-LibC from the Source Code
+
+AVR-LibC depends on [GNU Binutils](https://sourceware.org/binutils/) (>= 2.13)
+and [GCC](https://gcc.gnu.org/) (>= 3.3) that should be built for the AVR
```

Comment:
While these conditions on the versions are necessary, they are not sufficient.  For example, libc won't build with avr-gcc v3.4.  However such notes are usually understood as sufficient conditions, which they are not.

Though I don't know what's the oldest versions of the tools with which libc will work /build.  Definitely I wouldn't recommend anything below avr-gcc v4.7.

Reply:
Removed direct references to minimum supported versions. I think it makes more sense to keep and maintain these level of details in a separate page .